great_expectations/render/renderer_configuration.py

```diff
@@ -107,47 +112,23 @@ def _validate_configuration_or_result(cls, values: dict) -> dict:
             )
         return values
 
-    @root_validator()
-    def _validate_for_expectation_type_and_kwargs(cls, values: dict) -> dict:
-        if (
-            "result" in values
-            and values["result"] is not None
-            and values["result"].expectation_config is not None
-        ):
-            values["expectation_type"] = values[
-                "result"
-            ].expectation_config.expectation_type
-            values["kwargs"] = values["result"].expectation_config.kwargs
-        else:
-            values["expectation_type"] = values["configuration"].expectation_type
-            values["kwargs"] = values["configuration"].kwargs
-        return values
-
-    @root_validator()
-    def _validate_for_include_column_name(cls, values: dict) -> dict:
-        if "runtime_configuration" in values and values["runtime_configuration"]:
-            values["include_column_name"] = (
-                False
-                if values["runtime_configuration"].get("include_column_name") is False
-                else True
-            )
-        return values
-
     def __init__(self, **values) -> None:
         values["params"] = _RendererParamsBase()
         super().__init__(**values)
 
     class _RendererParamBase(BaseModel):
```

Comment:
`_RendererParamBase` and `_RendererParamsBase`?
Could you add a one or 2 line docstring to these classes to explain the difference?

Reply:
I added the docstrings. I'm looking into making this a custom root type if possible in the subsequent PRs.